scripts/server_scripts/mock_health_event_publisher.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+"""Module helps in generating the mock health event and publishing it to the message bus."""
+# This script is provided by HA. If any changes are done by HA Dev team, script needs to be
```

Comment:
why not add this as part of doc string itself.

Reply:
I will add this line inside doc string.